bootstrap.d/media-libs.yml

```diff
@@ -276,8 +276,7 @@ packages:
         - '-DCMAKE_INSTALL_DOCDIR=/usr/share/doc/libjpeg-turbo-2.0.5'
         - '-DCMAKE_INSTALL_DEFAULT_LIBDIR=lib'
         - '-DWITH_JPEG8=ON'
-        # HACK!! Hardcoding this makes it non-portable
-        - '-DCMAKE_SYSTEM_PROCESSOR=amd64'
+        - '-DCMAKE_SYSTEM_PROCESSOR=@OPTION:arch@'
```

Comment:
Is arm64 equivalent to x86_64 for this purpose? (The new option is set to x86_64.)

Reply:
Yes, libjpeg-turbo's CMake file accepts both x86_64 and amd64